packages/browser-runtime-core/src/interpreter/interpreter.spec.ts

```diff
@@ -159,24 +159,24 @@ describe('Interpreter', () => {
       ).to.equal(3);
     });
 
-    it.skip('can redeclare a top level function as function', async() => {
-      expect(
-        await testEvaluate(
-          'function f() { return 1; }',
-          'function f() { return 2; }',
-          'f()'
-        )
-      ).to.equal(2);
-    });
-
-    it.skip('can redeclare a top level function as var', async() => {
-      expect(
-        await testEvaluate(
-          'function sum(a, b) { return a + b; }',
-          'var sum = 1'
-        )
-      ).to.equal(1);
-    });
+    // it('can redeclare a top level function as function', async() => {
```

Comment:
this may really be an issue, again a corner case but this looks more realistic:

```
function f() { return 1; }
[Function: f]
function f() { return 2; }
[Function: f]
f()
1
```

Expected:

```
function f() { return 1; }
[Function: f]
function f() { return 2; }
[Function: f]
f()
2
```

Reply:
I think it's worth a ticket, but will also be closed when we do the unify eval epic so I'm happy to keep this as a skip so we're reminded that there's some TODO there